Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/VmServiceWrapper.java

```diff
@@ -198,6 +198,12 @@ public void addBreakpoint(@NotNull final String isolateId,
                             @NotNull final XLineBreakpoint<XBreakpointProperties> xBreakpoint,
                             @NotNull final VmServiceConsumers.BreakpointConsumerWrapper consumer) {
     final XSourcePosition position = xBreakpoint.getSourcePosition();
+    addBreakpointFromPosition(isolateId, position, consumer);
+  }
+
+  public void addBreakpointFromPosition(@NotNull final String isolateId,
+                                        @NotNull XSourcePosition position,
+                                        @NotNull final VmServiceConsumers.BreakpointConsumerWrapper consumer) {
     if (position == null || position.getFile().getFileType() != DartFileType.INSTANCE) {
```

Comment:
(looks like my inspections are working correctly, I just didn't notice this one)